src/wallet/init.cpp

```diff
@@ -16,6 +16,8 @@
 std::string GetWalletHelpString(bool showDebug)
 {
     std::string strUsage = HelpMessageGroup(_("Wallet options:"));
+    strUsage += HelpMessageOpt("-addresstype", strprintf(_("What type of addresses to use (\"legacy\", \"p2sh\", \"bech32\", default: \"%s\")"), FormatOutputType(OUTPUT_TYPE_DEFAULT)));
+    strUsage += HelpMessageOpt("-changetype", strprintf(_("What type of change to use (\"legacy\", \"p2sh\", \"bech32\", default is same as -addresstype: \"%s\")"), FormatOutputType(OUTPUT_TYPE_DEFAULT)));
```

Comment:
This is ever so slightly confusing, maybe skip the FormatOutputType part and just say "default is whatever is set for -addresstype".

Reply:
Will fix.